src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -48,13 +44,31 @@ public CommandResult execute() {
      */
     private List<ReadOnlyPerson> getPersonsWithNameContainingAnyKeyword(Set<String> keywords) {
         final List<ReadOnlyPerson> matchedPersons = new ArrayList<>();
+        List<String> nameSplitByWhiteSpace = new ArrayList<>();
```

Comment:
Variables should be declared in the smallest scope possible.
Please see https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html#variables

Can some variable declarations be moved inside the for loop?